tests/countries/test_american_samoa.py

```diff
@@ -13,31 +13,28 @@
 from unittest import TestCase
 
 from holidays.constants import GOVERNMENT, UNOFFICIAL
-from holidays.countries.american_samoa import HolidaysAS, AS, ASM
+from holidays.countries.american_samoa import HolidaysAS, AmericanSamoa, AS, ASM
 from tests.common import CommonCountryTests
 
 
 class TestAS(CommonCountryTests, TestCase):
     @classmethod
     def setUpClass(cls):
-        super().setUpClass(HolidaysAS)
+        super().setUpClass(HolidaysAS, years=range(1901, 2050))
```

Comment:
There is no need to initialize the entire year range, because we don't have tests for it.

Reply:
Fixed